pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nornir-infrahub"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
opinion: should we bump version to 1.0 here as well to keep it consistent with sdk / infrahub?

Reply:
Actually good point. This version is probably not backwards compatible so we should bump the major version.
It doesn't have to be consistent with the SDK/Infrahub though.